README.md

```diff
@@ -1,24 +1,124 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🗝️ Key Features](#key-features)
+  - [🛠️ Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 Hello Rails React <a name="about-project"></a>
 
-* System dependencies
+**Hello Rails React** is an exercise to setup RoR + React as one app with Webpack.
 
-* Configuration
+## 🗝️ Key Features <a name="key-features"></a>
 
-* Database creation
+- Implements `jsbundling-rails`
+- Implements `react-router`
+- Implements `redux`
 
-* Database initialization
+## 🛠️ Built With <a name="built-with"></a>
 
-* How to run the test suite
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev">React</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+- Install the [rails gem](https://rubygems.org/gems/rails)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:dicodiaz/hello-rails-react.git
+```
+
+### Usage
+
+To run the project, execute the following commands:
+
+```sh
+  rails server
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Dico Diaz Dussan**
+
+- GitHub: [@dicodiaz](https://github.com/dicodiaz)
+- LinkedIn: [Dico Diaz Dussan](https://www.linkedin.com/in/dico-diaz-dussan/)
+- Portfolio: [portfolio.dicodiaz.com.co](https://portfolio.dicodiaz.com.co)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+No future features are currently planned
```

Comment:
- [ ] It would be better if we replace this text with a more concrete feature, like for example `Improve styling of the app`. This can also be considered a feature so let us kindly take note